plugins/backend/rag-ai-backend-embeddings-openai/README.md

```diff
@@ -30,7 +30,10 @@ ai:
     # OpenAI Embeddings configuration
     openai:
       # (Optional) The API key for accessing OpenAI services. Defaults to process.env.OPENAI_API_KEY
-      openAIApiKey: 'sk-123...'
+      openAiApiKey: 'sk-123...'
+
+      # (Optional) Specify URL of self-hosted OpenAI compliant endpoint. Defaults to OpenAI's public API https://api.openai.com
```

Comment:
Please mention what the default state is.

Reply:
is this not sufficient? `Defaults to OpenAI's public API https://api.openai.com`

Reply:
Ah sorry. Somehow i missed that.

Reply:
Ah sorry. Somehow i missed that.

Reply:
No worries!